qa/tasks/cephfs/test_volumes.py

```diff
@@ -1029,9 +1030,12 @@ def test_subvolume_create_with_desired_mode_in_group(self):
         subvol3_path = self._get_subvolume_path(self.volname, subvol3, group_name=group)
 
         # check subvolume's  mode
-        actual_mode1 = self.mount_a.run_shell(['stat', '-c' '%a', subvol1_path]).stdout.getvalue().strip()
```

Comment:
That's all just a code duplication, try this:
```
def get_actual_mode(self, path):
   mode = self.mount_a.client_remote.sh(['sudo', 'stat', '-c', '%s', path]).strip()
   return mode
.
.
.
actual_mode1 = self.get_actual_mode(subvol1_path)
. . .
````

Reply:
Will try to fix the s/run_shell/remote.sh stuff in one separate PR and together with many other enhancements in vstart_runner.py later.